provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/parser/X509RegistrationResultParser.java

```diff
@@ -24,7 +24,7 @@
      * Class representing X509CertificateInfo
      * https://docs.microsoft.com/en-us/rest/api/iot-dps/RuntimeRegistration/RegisterDevice#definitions_x509certificateinfo
      */
-    public class X509CertificateInfo
+    public static class X509CertificateInfo
```

Comment:
@timtay-microsoft is this something users initialize?

Reply:
Nope. We initialize it internally for DPS, but users never make an instance of it